osu.Game/Screens/Edit/Compose/Components/ComposeBlueprintContainer.cs

```diff
@@ -295,8 +295,11 @@ private void refreshTool()
             ensurePlacementCreated();
         }
 
-        private void updatePlacementPosition()
+        private void updatePlacementTimeAndPosition()
         {
+            if (CurrentPlacement == null)
+                return;
```

Comment:
I assume this was added specifically for the new usage of this in `Update()`? Can this conditional be moved there?

Reply:
In `OnMouseMove()`, but yeah sure, applied in https://github.com/ppy/osu/pull/30263/commits/40b98d48632b1a051faa19b51ddf666aa1d219e6.

Reply:
Apparently tests disagree with that.